datacenter/ucp/2.1/guides/admin/configure/content-trust/admin_tasks.md

```diff
@@ -0,0 +1,387 @@
+    >  and destination directory will be included in the command automatically.
+    {: id="notary_alias_config_note" }
+
+3.  Find the _globally unique name (GUN)_ for your repository. The GUN
+    is the `<registry/<account>/<repository>` string, such as
+    `dtr-example.com/engineering/my-repo`. You can find the GUN for a repository
+    by browsing to it in the DTR web UI and copying the part of the
+    **Pull command** after `docker pull`.
+
+Next, [initialize a trusted image repository](#set-up-a-trusted-image-repository).
+
+## Initialize the trusted image repository
+
+> **Tip - Yubikey integration**: Notary supports integration with Yubikey. If
+> you have a Yubikey plugged in when you initialize a repository with Notary,
+> the root key is stored on the Yubikey instead of in the trust directory. When
+> you run any command that needs the `root` key, Notary looks on the Yubikey
+> first, and uses the trust directory as a fallback.
+
+This procedure needs to be done on the client of an administrator responsible for
+setting up repositories. It needs to be done once per signing repository.
+
+In these examples, add the `-s`, `-d`, and `--tlscacert` parameters you need
+before `<GUN>` if you decided not to configure Notary using a
+[configuration file](/notary/reference/client-config.md){: target="_blank" class="_" }
+or a [terminal alias](#notary_alias_config_note").
+
+1.  In the DTR web UI, create a new repository or browse to an existing
+    repository that you want to reconfigure as a trusted image repository.
+    Make a note of the GUN for the repository by copying the contents of the
+    **Pull command** field after `docker pull`.
+
+    ![Finding the GUN in the DTR UI](/datacenter/images/dtr_repo_find_gun.png)
+
+2.  At the command line on your client, check whether Notary has information
+    about the repository. Most likely if you are performing this task for the
+    first time, the repository is not initialized.
+
+    ```bash
+    $ notary list <GUN>
+    ```
+
+    The response may be one of the following:
+
+    - `fatal: client is offline`: Either the repository server can't be reached,
+      or DTR is using certificates which are not signed by a globally trusted
+      certificate authority, such as self-signed certificates. Run `notary list`
+      again, adding the `--tlscacert` flag, with the path to the certificate
+      authority for DTR. To get the certificate, download `https://<dtr_url>/ca`
+      from your browser or using `curl` or `wget`. This certificate is
+      different from the UCP trust certificate in the UCP client bundle.
+
+    - `fatal: <dtr_url> does not have trust data for ddc-staging-dtr.qa.aws.dckr.io/engineering/redis`:
+      The repository has not yet been initialized and you need to run
+      `notary init`. Continue to step 2.
+
+    - `No targets present in this repository.`: The repository has been
+      initialized, but contains no signed images. You do not need to do step 2.
```

Comment:
Maybe in this and the next item, rather than saying that step 2 can be skipped (because step 3 can also be skipped) the user can be linked to the "Delegate image signing" step instead?